specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/firewallRule.json

```diff
@@ -172,7 +172,7 @@
         "responses": {
           "200": {
             "schema": {
-              "type": "object"
+              "$ref": "#/definitions/IpFirewallRuleInfo"
```

Comment:
I am curious why
1. Delete operation has a response with schema?
2. this is clearly a breaking change, our breaking change pipeline says nothing? Is it because this is a LRO?

Reply:
1. Resolved this issue for Invalid_Type.
2. I looked into example and updated the schema reference. Unaware of your ask